plugin-modernizer-cli/src/main/java/io/jenkins/tools/pluginmodernizer/cli/PluginLoggerDiscriminator.java

```diff
@@ -11,10 +15,15 @@ public class PluginLoggerDiscriminator extends AbstractDiscriminator<ILoggingEve
     public String getDiscriminatingValue(ILoggingEvent iLoggingEvent) {
         List<Marker> markers = iLoggingEvent.getMarkerList();
         if (markers == null || markers.isEmpty()) {
-            return "modernizer";
+            return Path.of(System.getProperty("user.home"), ".cache", "jenkins-plugin-modernizer-cli", "modernizer.logs").toString();
```

Comment:
The problem is that is doesn't honor the Settings and Config for the --cache-path option.

Did some test where done with such option ?

I think we also need to update the integration test that create attachement for such logs